### PR TITLE
Fix label size

### PR DESCRIPTION
**Pull request by evmosseveld**

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
@evmosseveld what tape sizes have you tested this on?

**Comment by evmosseveld** [comment]
> @evmosseveld what tape sizes have you tested this on?

Currently I only have the 29mm x 90mm die-cut tape to test it with, so I don't know if it works for all labels. The problem I was having was that the label wouldn't print because the brother_ql library only accepts the image if it's precisely the correct size (at least for my media type, but I suspect it's the same for all die-cut tapes). So I needed to resize the image (as stated in the code ideally there would be a GUI to configure this).

**Comment by evmosseveld** [comment]
Btw I was wondering which things were meant by these 2 TODOs:
```
# TODO: Add padding around the provided image, otherwise the label does not print correctly
# TODO: Improve label auto-scaling based on provided width and height information
```
The resizing I have now implemented, is that what was meant by 'padding' in the TODO?
The same goes for the auto-scaling (although the 'provided width and height' that we get are those of the label template before conversion to PDF and PNG so I don't know how useful that information would be).

File: inventree_brother/brother_plugin.py [hunk 1]
```diff
@@ -104,22 +106,39 @@ def print_label(self, **kwargs):
...
+        media_type = self.get_setting('LABEL')
+
+        # As the brother_ql library only accepts an image with exactly the right size for the label type,
+        # we need to resize the image. For now just center the image with some whitespace around the side,
+        # but ideally this plugin or maybe the label printing base module should provide some formatting
+        # options for scaling, margins, etc.
+        # TODO: correctly handle case where label image is bigger than printable area
+        for media_specifications in ALL_LABELS:
+            if media_specifications.identifier == media_type:
+                printable_dimensions = media_specifications.dots_printable
```
Comment (SchrodingersGat) [inline inventree_brother/brother_plugin.py:134]: You will want to handle the case where `printable_dimensions` is not set here

File: inventree_brother/brother_plugin.py [hunk 2]
```diff
@@ -104,22 +106,39 @@ def print_label(self, **kwargs):
...
+        printable_image = Image.new("RGB", printable_dimensions, "white")
+        margin_left = (printable_dimensions[0] - label_image.width) // 2
```
Comment (SchrodingersGat) [inline inventree_brother/brother_plugin.py:136]: Should handle the case where these calculations would result in negative values

**Comment by SchrodingersGat** [comment]
@evmosseveld thanks for the improvements, I'll push a new release

**Pull request merged**